src/nvim/tui/tui.c

```diff
@@ -583,6 +588,17 @@ static void tui_flush(UI *ui)
   TUIData *data = ui->data;
   UGrid *grid = &data->grid;
 
+
+  int nrevents = queue_count(data->loop->thread_events);
+  // QUEUE *q = QUEUE_HEAD(&queue->headtail);
+  // while(true) {
+  // }
+  ILOG("thread_events=%d", nrevents);
```

Comment:
Yep, I'll add a condition or remove this. `yes` can generate MB of data in a couple seconds, and GB in just a few minutes.